.s2i/httpd-cfg/01-commercial.conf

```diff
@@ -139,6 +139,8 @@ AddType text/vtt                            vtt
     # it should probably be best to combine the next few lines in one reg exp but for clarity keeping them separate
     # the first one redirects
     RewriteRule ^acs/?$ /acs/3.69/welcome/index.html [R=301]
+    # redirect to the latest release notes
+    RewriteRule ^acs/release_notes/$ /acs/3.69/release_notes/369-release-notes.html? [R=301,NE,NC,L]
```

Comment:
@gaurav-nelson the redirect rule looks good to me.

Is there a specific reason for including the `NC` and `L` flags in this case?

Reply:
@pneedle-rh nope, I've removed those to match with the other rules.